### DISCUSSION TITLE
Backgroun-image invisible

### DISCUSSION THREAD

**Discussion by kvnhit** [darkreader/darkreader#11188] [Q&A]
Hello, everyone

I have a web application where, when I enable dark mode, the background-images of select elements become invisible. It seems that DarkReader is replacing them with another image that is not visible. How can I solve this issue?

**Comment by ngoclong19** [comment]
Could you try to debug using the steps outline in this comment https://github.com/darkreader/darkreader/issues/11069#issuecomment-1500880164

Maybe we have the same issue.
